common/djangoapps/student/views/dashboard.py

```diff
@@ -472,6 +472,24 @@ def get_dashboard_course_limit():
     return course_limit
 
 
+def check_for_unacknowledged_notices(context):
+    """
+    Checks the notices apps plugin context to see if there are any unacknowledged notices the user needs to take action
+    on. If so, build a redirect url to the first unack'd notice.
+    """
+    notice_url = None
+
+    if context.get("plugins", {}).get('notices'):
+        notices_context = context.get("plugins").get("notices")
+        if notices_context.get("unacknowledged_notices"):
+            notices = notices_context.get("unacknowledged_notices")
```

Comment:
This seems a bit repetitive. Can we just collapse them to a:
```
notices = context.get("plugins", {}).get("notices", {}).get("unacknowledged_notices")
if notices:
   # do thing
```

Reply:
Sure, makes sense to me.